### PR TITLE
Release Statemine v5

### PR DESCRIPTION
**Pull request by apopiak**
**Note: Don't Merge**

WIP release candidate branch for Statemine v5 (based on `polkadot-v0.9.11`)

### CODE REVIEW

File: polkadot-parachains/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "polkadot-collator"
-version = "0.1.0"
+version = "4.0.0"
```
Comment (chevdor) [inline polkadot-parachains/Cargo.toml:3]:
```suggestion
version = "5.0.0"
```

**Comment by chevdor** [comment]
The `srtool` build issue was due only to the rococo runtime not building. It has been fixed in 4a704cc593134bb4ec7336427e9b54f79398d3ea

File: polkadot-parachains/statemine/src/weights/pallet_utility.rs [hunk 2]
```diff
@@ -42,20 +42,20 @@
...
-		(4_979_000 as Weight)
+		(5_201_000 as Weight)
 	}
 	fn batch_all(c: u32, ) -> Weight {
-		(36_922_000 as Weight)
-			// Standard Error: 3_000
-			.saturating_add((8_534_000 as Weight).saturating_mul(c as Weight))
+		(66_893_000 as Weight)
+			// Standard Error: 6_000
+			.saturating_add((7_533_000 as Weight).saturating_mul(c as Weight))
```
Comment (chevdor) [inline polkadot-parachains/statemine/src/weights/pallet_utility.rs]: Noticeable increase

Comment (chevdor) [reply polkadot-parachains/statemine/src/weights/pallet_utility.rs]: This is the only one I spotted

File: polkadot-parachains/statemine/src/weights/pallet_utility.rs [hunk 3]
```diff
@@ -15,28 +31,31 @@
...
+		(5_201_000 as Weight)
 	}
-	fn batch_all(c: u32) -> Weight {
-		(17_745_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((4_578_000 as Weight).saturating_mul(c as Weight))
+	fn batch_all(c: u32, ) -> Weight {
+		(66_893_000 as Weight)
+			// Standard Error: 6_000
+			.saturating_add((7_533_000 as Weight).saturating_mul(c as Weight))
```
[the remark this answers is not in the document]
Comment (NachoPal) [reply polkadot-parachains/statemine/src/weights/pallet_utility.rs:59]: For `statemint`, compared to the values you can find in substrate, it is almos the double though `46_068_000` vs https://github.com/paritytech/substrate/blob/polkadot-v0.9.11/frame/utility/src/weights.rs#L65

**Comment by apopiak** [comment]
we released :tada: https://github.com/paritytech/cumulus/releases/tag/statemine-v5.0.0

**Pull request closed without merging**